docs/news.rst

```diff
@@ -32,6 +32,8 @@ Psycopg 3.2.8 (unreleased)
 - Don't process further connection attempts after Ctrl-C (:ticket:`#1077`).
 - Fix cursors to correctly iterate over rows even if their row factory
   returns `None` (:ticket:`#1073`).
+- Add `~Cursor.sets()` to iterate over the result sets of the queries
+  executed though `~Cursor.executemany()` or `~Cursor.execute()`.
```

Comment:
This change would be released in 3.3 because it is a new feature. You mat mention ticket #1080, where you proposed the idea in first place.